### DISCUSSION TITLE
LVM module on netdata 1.45.4 is not loaded

### DISCUSSION THREAD

**Discussion by vgeorgik** [netdata/netdata#17625] [Q&A]
Hello,

I have an oracle linux 8.9 with latest netdata, 1.45.4 ( and i am not able to load the lvm module following the official guide.

Please not that Selinux is on enforcing mode.

The result is that i cannot get disk_util.vg metrics.

On the debug, i am getting **"no module to run component=agent"** 

<img width="627" alt="netdata_lvm" src="https://github.com/netdata/netdata/assets/91597723/9b9a8567-6a45-4502-8f7b-ca489d9c1043">

Installed netdata via kickstart.sh

Am i missing something?

thanks in advance,
Vasilis

**Comment by ilyam8** [comment]
Hi, @vgeorgik. It is not in v1.45.4 (patch release), it will be in v1.46.0 (next minor release).

**Reply by vgeorgik** [reply]
Hi @ilyam8, 
Thank you for the fast reply.
On previous version, e.g 1.34, i was able to get disk_util.vg metrics. 
How come this was not included in the newest versions?

**Reply by ilyam8** [reply]
This has nothing to do with the `go.d/lvm` collector.

disk_util metric is collected by the proc plugin by reading `/proc/diskstats`. Netdata collects disk statistics metrics for all disks except those that match the "exclude disks" pattern.

<details>
<summary>Check "Top->Block-devices".</summary>

<img width="1658" alt="Screenshot 2024-05-09 at 15 26 44" src="https://github.com/netdata/netdata/assets/22274335/9fcfba29-31cd-4c90-a3d3-008c1569722b">

</details>

**Comment by vgeorgik** [comment]
i cannot find any block-devices section/chart

![image](https://github.com/netdata/netdata/assets/91597723/9a543410-37e1-4496-88ac-d969ad4ed5e4)

**Reply by ilyam8** [reply]
Because there is none. Did you check my screenshot? It is “Top” tab.

**Comment by vgeorgik** [comment]
Yes i did but i dont have any Functions in the dashboard

**Comment by vgeorgik** [comment]
sorry didn't get to "Top" tab part.
Will check and get back.
